internal/bminventory/inventory.go

```diff
@@ -85,6 +85,7 @@ type Config struct {
 	SkipCertVerification bool              `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
 	InstallRHCa          bool              `envconfig:"INSTALL_RH_CA" default:"false"`
 	RhQaRegCred          string            `envconfig:"REGISTRY_CREDS" default:""`
+	AgentTimeoutStartSec int64             `envconfig:"AGENT_TIMEOUT_START_SEC" default:"300"`
```

Comment:
This should be time.Duration with a default of "3m"
See ImageExpirationTime variable above

Reply:
OK

Reply:
This is the way that we specify timeouts to the asssited-service
See how `AgentTimeoutStartSec` is used